### PR TITLE
doc: update descriptions about nydus-snapshotter

### PR DESCRIPTION
**Pull request by changweige**
To match the latest nydus-snapshotter UI

### CODE REVIEW

File: docs/containerd-env-setup.md [hunk 1]
```diff
@@ -132,9 +134,9 @@ sudo docker run -d --restart=always -p 5000:5000 registry
 
 ## Convert an Image to Nydus Format
 
-Currently, nydus image must be created by converting from an existed OCI or docker v2 image.
+Nydus image can be created by converting from an existed OCI or docker v2 image stored in container registry or directly built from Dockerfile(with Buildkit)
```
Comment (jiangliu) [inline docs/containerd-env-setup.md:137]: existed -> existing?

Comment (changweige) [reply docs/containerd-env-setup.md]: fixed. thanks

File: docs/containerd-env-setup.md [hunk 2]
```diff
@@ -74,21 +74,23 @@ sudo rm -rf /var/lib/containerd-nydus
 ```
 
 3. Start `containerd-nydus-grpc` (nydus snapshotter):
+Optionally, a TOML nydus-snapshotter configuration file can be provided by appending `--config <CONFIG>` when starting nydus-snapshotter if you want fine-grained control items. An example configuration file can be found [here](https://github.com/containerd/nydus-snapshotter/blob/main/misc/snapshotter/config.toml)
```
Comment (imeoer) [inline docs/containerd-env-setup.md:77]: `a TOML based`?

File: docs/containerd-env-setup.md [hunk 3]
```diff
@@ -74,21 +74,23 @@ sudo rm -rf /var/lib/containerd-nydus
...
 
 ## Configure and Start Containerd
 
-Nydus uses two features of containerd:
+Nydus depends on two features of Containerd:
 
-- remote snapshotter
-- snapshotter annotations
+- Remote snapshotter plugin support
+- Pass the annotations of image layer to the remote snapshotter
```
Comment (imeoer) [inline docs/containerd-env-setup.md:91]: - Support remote snapshotter plugin
- Support passing image-related annotations to remote snapshotter

File: docs/containerd-env-setup.md [hunk 4]
```diff
@@ -132,9 +134,9 @@ sudo docker run -d --restart=always -p 5000:5000 registry
 
 ## Convert an Image to Nydus Format
```
Comment (imeoer) [inline docs/containerd-env-setup.md:135]: `Convert/Build`

File: docs/containerd-env-setup.md [hunk 5]
```diff
@@ -132,9 +134,9 @@ sudo docker run -d --restart=always -p 5000:5000 registry
...
 
-Currently, nydus image must be created by converting from an existed OCI or docker v2 image.
+Nydus image can be created by converting from an existing OCI or docker v2 image stored in container registry or directly built from Dockerfile(with Buildkit)
```
Comment (imeoer) [inline docs/containerd-env-setup.md:137]: Can we give the buildkit doc link: https://github.com/moby/buildkit/blob/master/docs/nydus.md

**Review by imeoer — COMMENTED** [review]
Thanks!

File: docs/containerd-env-setup.md [hunk 6]
```diff
@@ -74,21 +74,23 @@ sudo rm -rf /var/lib/containerd-nydus
 ```
 
 3. Start `containerd-nydus-grpc` (nydus snapshotter):
+Optionally, a TOML nydus-snapshotter configuration file can be provided by appending `--config <CONFIG>` when starting nydus-snapshotter if you want fine-grained control items. An example configuration file can be found [here](https://github.com/containerd/nydus-snapshotter/blob/main/misc/snapshotter/config.toml)
 
 ```bash
 sudo /usr/bin/containerd-nydus-grpc \
-    --config-path /etc/nydus/nydusd-config.fusedev.json \
+    --fs-driver fusedev \
```
Comment (imeoer) [inline docs/containerd-env-setup.md:81]: Can we make it as default?

Comment (imeoer) [reply docs/containerd-env-setup.md:81]: It seems the latest snapshotter must be specified `--address /run/containerd-nydus/containerd-nydus-grpc.sock`.

Comment (adamqqqplay) [reply docs/containerd-env-setup.md:81]: @changweige By the way, when this API changes in nydus-snapshotter, can we give a detailed description when the version is released? Recent API changes are somewhat frequent.

**Review by ccx1024cc — COMMENTED** [review]
Will this pr backport to stable/v2.2 ?

File: docs/containerd-env-setup.md (hunk 6 above)
Comment (changweige) [reply docs/containerd-env-setup.md:81]: > It seems the latest snapshotter must be specified `--address /run/containerd-nydus/containerd-nydus-grpc.sock`.

It has been fixed

File: docs/containerd-env-setup.md (hunk 4 above)
Comment (changweige) [reply docs/containerd-env-setup.md]: fixed

**Comment by changweige** [comment]
> Will this pr backport to stable/v2.2 ?

No. It's fine for documents to only update on master branch

File: docs/containerd-env-setup.md (hunk 3 above)
Comment (changweige) [reply docs/containerd-env-setup.md]: fixed

File: docs/containerd-env-setup.md (hunk 2 above)
Comment (changweige) [reply docs/containerd-env-setup.md]: fixed

**Review by imeoer — APPROVED** [review]

**Pull request merged**